### PR TITLE
Add ranking to projects

### LINKED ISSUE #1161: Add Rank Order to Projects
So we can can control the order they appear on the page.

### PR DESCRIPTION
**Pull request by edpaget**
When projects are requested with beta_approved or launch_approved they
will be sorted by default by their assigned ranking.

Closes #1161

### CODE REVIEW

**Comment by camallen** [comment]
LGTM - are you going to set some defaults?

**Comment by edpaget** [comment]
Not in code. I was going to go through and at least order the launched panoptes projects above the rest of them.

**Comment by edpaget** [comment]
Ah the failures seem to be due to rails' awesome idea to convert params to strings in controller tests :+1:

File: spec/support/admin_only_option.rb [hunk 1]
```diff
@@ -10,6 +11,7 @@
   context "when the user is an admin" do
     let(:authorized_user) { create(:admin_user) }
     it "should create the project" do
+      p json_response
```
Comment (camallen) [inline spec/support/admin_only_option.rb]: might want to remove this

**Comment by camallen** [comment]
This looks good to go.

**Pull request merged**